news/2023/2023-11-18-new-featured-artist-you.md

```diff
@@ -0,0 +1,67 @@
+
+**you**'s goal is simple: "to create sounds that are fun to listen to".
+
+While **you** is nearly impossible to find through a Google search, their style has already reached countless corners of the internet! From eSports organisations like *Crazy Raccoon* and *Yuki Aim* to TV shows, PC/mobile games, idols, VTubers, and (of course) other rhythm games, it's impossible to hide from **you**.
+
+**10** tunes by **you** are available on [this Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/389), pre-timed and ready for... someone.
+
+Curious how **you** would fit in a beatmap? Treat yourself to 4 of them by watching this osu!taiko-themed video or by playing the maps below!
+
+<div align="center">
+    <video width="85%" controls>
+        <source src="https://assets.ppy.sh/artists/389/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### you - Hold Angel
+
+Check out [this legendary 7K osu!mania map](https://osu.ppy.sh/beatmapsets/636543) hosted by [Blocko](https://osu.ppy.sh/users/4075092) and ranging from 1 to 10 stars, or try [this osu!taiko rendition](https://osu.ppy.sh/beatmapsets/381103) hosted by [Nishizumi](https://osu.ppy.sh/users/2496768)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/389/Songs/you - Hold Angel.mp3">
+</audio>
+
+### you - Wake Up
+
+Set your alarm to the song behind [this 42 second osu!taiko map](https://osu.ppy.sh/beatmapsets/592892) hosted by [komasy](https://osu.ppy.sh/users/1980256)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/389/Songs/you - Wake Up.mp3">
+</audio>
+
+### you - Diamond Smile (2019 Reworks)
+
+Take your pick: [this osu!taiko marathon](https://osu.ppy.sh/beatmapsets/904130) hosted by [komasy](https://osu.ppy.sh/users/1980256) or [this other osu!taiko marathon](https://osu.ppy.sh/beatmapsets/363525) hosted by [Nanatsu](https://osu.ppy.sh/users/2786984)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/389/Songs/you feat. nayuta - Diamond Smile (2019 Reworks).mp3">
+</audio>
+
+### you - Let's Jump (2019 Reworks)
+
+Jump into [this osu!taiko map](https://osu.ppy.sh/beatmapsets/1108981) hosted by [Nanatsu](https://osu.ppy.sh/users/2786984)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/389/Songs/you feat. nayuta - Let_s Jump! (2019 Reworks).mp3">
+</audio>
+
+---
+
+You should head to [**you**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/389) where you can find beatmap templates for all of **you**'s tracks! If "you" still looks like a real word to you by now, then you clearly haven't read "you" enough times.
+
+In case you forgot, there's a batch of [**Featured Artist playlists** in osu!(lazer)](/wiki/People/Featured_Artists/Featured_Artist_playlists) running right now! You have a week til score submission closes, so do what you have to do.
```

Comment:
```suggestion
In case you forgot, there's a batch of [**Featured Artist playlists** in osu!(lazer)](/wiki/People/Featured_Artists/Featured_Artist_playlists) running right now! You have a week 'til score submission closes, so do what you have to do.
```